spec/order_spec.rb

```diff
@@ -0,0 +1,46 @@
+require 'order'
+
+describe Order do
+  let(:stew_dish) { double(:stew, name: :stew, price: 2) }
+  let(:rice_dish) { double(:rice, name: :rice, price: 1) }
+  let(:list_of_dishes) { double(:dish_list) }
+  let(:selections) { { stew: 2, rice: 1 } }
+
+  before do
+    allow(list_of_dishes).to receive(:get_dish).with(:stew).and_return(stew_dish)
+    allow(list_of_dishes).to receive(:get_dish).with(:rice).and_return(rice_dish)
+  end
+
+  context 'functionality' do
+    it 'shows an order summary' do
+      order = Order.new(list_of_dishes, selections, 5)
+      expect(order.summary).to eq({ stew: { quantity: 2, price: 2 }, rice: { quantity: 1, price: 1 } })
+    end
+
+    it 'sends a text with the expected delivery time when an order has been verified' do
+      text_handler = double(:text_handler, send_confirmation_text: 'Text sent')
+      order = Order.new(list_of_dishes, selections, 5)
+      expect(order.send_text(text_handler)).to eq('Text sent')
```

Comment:
I notice that the unit test for `send_confirmation_text()` does not expect it to return anything.

Maybe `send_confirmation_text()` does not need to return anything when it succeeds? We could just expect the method to be called?